kf_lib_data_ingest/network/utils.py

```diff
@@ -245,3 +250,58 @@ def get_open_api_v2_schema(url, entity_names,
             print(err)
 
     return output
+
+
+def get_tag_commit_sha(tag_name,
```

Comment:
If you can write the tag in a file, why can't you just write down the commit hash for the tag and not have to go fetch it?

Reply:
True, we could do away with this function if something else (maybe the release process) is able to generate an updated release tag, commit hash file when a new release is made. Then this file does become redundant with `__init__.py` bc we can look at the latest/last entry in the file.

Reply:
Its not a standard, but I read that people typically store the version of a python package in __version__. I've seen that stored in the package's __init__.py, or some other file. Maybe for us, since we will always want to know the commit hash as well as the version, we should just store the mapping file of the tags and hashes?

Reply:
> Does this mean you want to update this file every release?

Yea, probably (and then as I mentioned above, we would get rid of `__init__.py` w/ the version